strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/cron/jobs/MergeMavenGroupRepositoryIndexCronJob.java

```diff
@@ -0,0 +1,75 @@
+package org.carlspring.strongbox.cron.jobs;
+
+import org.carlspring.strongbox.configuration.ConfigurationManager;
+import org.carlspring.strongbox.cron.domain.CronTaskConfigurationDto;
+import org.carlspring.strongbox.cron.jobs.fields.*;
+import org.carlspring.strongbox.storage.indexing.RepositoryIndexFounder;
+import org.carlspring.strongbox.storage.indexing.RepositoryIndexFounder.RepositoryIndexFounderQualifier;
+import org.carlspring.strongbox.storage.repository.Repository;
+import org.carlspring.strongbox.storage.repository.RepositoryTypeEnum;
+
+import javax.inject.Inject;
+import java.util.Set;
+
+import com.google.common.collect.ImmutableSet;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+public class MergeMavenGroupRepositoryIndexCronJob
+        extends JavaCronJob
+{
+
+    private static final String PROPERTY_STORAGE_ID = "storageId";
+
+    private static final String PROPERTY_REPOSITORY_ID = "repositoryId";
+
+    private static final Set<CronJobField> FIELDS = ImmutableSet.of(
+            new CronJobStorageIdAutocompleteField(new CronJobStringTypeField(
+                    new CronJobRequiredField(new CronJobNamedField(PROPERTY_STORAGE_ID)))),
+            new CronJobRepositoryIdAutocompleteField(new CronJobStringTypeField(
+                    new CronJobRequiredField(new CronJobNamedField(PROPERTY_REPOSITORY_ID)))));
+
+    @Inject
+    @RepositoryIndexFounderQualifier(RepositoryTypeEnum.GROUP)
```

Comment:
Cosmetic: Please rename this, as requested below. Thanks! :)

Reply:
done